Jenkinsfile

```diff
@@ -63,5 +63,28 @@ pipeline {
                 }
             }
         }
+        stage('Release: [ master ]') {
+            when {
+                allOf {
+                    branch 'master'
+                    not {
+                        changelog '.*\\[skip ci\\]$'
+                    }
```

Comment:
You need the NPM token to publish: check  env.NPM_TOKEN = sh(script: 'vault read -field=value secret/ops/token/npm', returnStdout: true), the vault container

Reply:
Added!